src/pages/onboarding/onboarding.tsx

```diff
@@ -1,5 +1,153 @@
+import Button from '@components/button/button/button';
+import { useFunnel } from '@hooks/use-funnel';
+import Complete from '@pages/onboarding/components/complete';
+import Gender from '@pages/onboarding/components/gender';
+import MatchingType from '@pages/onboarding/components/matching-type';
+import OnboardingHeader from '@pages/onboarding/components/onboarding-header';
+import ProgressBar from '@pages/onboarding/components/progress-bar';
+import Start from '@pages/onboarding/components/start';
+import SupportTeam from '@pages/onboarding/components/support-team';
+import SyncSupportTeam from '@pages/onboarding/components/sync-support-team';
+import ViewingStyle from '@pages/onboarding/components/viewing-style';
+import { FIRST_FUNNEL_STEPS } from '@pages/onboarding/constants/onboarding';
+import { ROUTES } from '@routes/routes-config';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
 const Onboarding = () => {
-  return <div>최초 매칭 조건 설정 페이지</div>;
+  const { Funnel, Step, currentStep, currentIndex, steps, goNext, goPrev } = useFunnel(
+    FIRST_FUNNEL_STEPS,
+    ROUTES.HOME,
+  );
+
+  const [selections, setSelections] = useState<Record<string, string | null>>({
+    SUPPORT_TEAM: null,
+    SYNC_SUPPORT_TEAM: null,
+    VIEWING_STYLE: null,
+    GENDER: null,
+    MATCHING_TYPE: null,
+  });
+
+  const handleSelect = (stepName: string, value: string) => {
+    setSelections((prev) => ({ ...prev, [stepName]: value }));
+  };
+
+  const isStepCompleted = (stepName: string) => selections[stepName] !== null;
+
+  const navigate = useNavigate();
+
+  return (
+    <div className="flex h-[100svh] flex-col">
+      <div className="sticky top-0 bg-background">
+        <OnboardingHeader onClick={goPrev} />
+        {currentStep !== 'START' && (
+          <div className="w-full">
+            <ProgressBar currentStep={currentIndex} totalSteps={steps.length - 1} />
+          </div>
+        )}
+      </div>
+
+      <div className="h-full flex-col-between">
+        <Funnel>
+          <Step name="START">
+            <Start />
+            <div className="onboarding-footer">
+              <Button label="시작하기" size={'L'} variant={'blue'} onClick={goNext} />
+            </div>
```

Comment:
버튼들은 아래에 하나의 Button만 두고 유틸함수들 빼서 관리하면 될 것 같아요 onClick 안에도 분기처리가 들어가면서 가독성이 떨어지는 느낌이라 ㅠㅠ

```suggestion
const getButtonLabel = () => {
    switch (currentStep) {
      case 'START':
        return '시작하기';
      case 'COMPLETE':
        return '메인 화면으로 이동하기';
      default:
        return '다음으로';
    }
  };

  const isButtonDisabled = () => {
    if (currentStep === 'START' || currentStep === 'COMPLETE') return false;
    return selections[currentStep] === null;
  };

  const handleButtonClick = () => {
    if (currentStep === 'START') {
      goNext();
    } else if (currentStep === 'MATCHING_TYPE') {
      if (selections.MATCHING_TYPE === '1:1 매칭') {
        goNext();
      } else {
        navigate(ROUTES.ONBOARDING_GROUP);
      }
    } else if (currentStep === 'COMPLETE') {
       navigate(ROUTES.HOME);
    } else {
      goNext();
    }
  };
```